src/screens/ProfileScreen/index.js

```diff
@@ -56,21 +49,28 @@ import {
   updateBioProfile,
   updateImageProfile
 } from '../../service/profile';
-import {colors} from '../../utils/colors';
-import {deleteAnonymousPost, deletePost} from '../../service/post';
 import {downVote, upVote} from '../../service/vote';
+import {colors} from '../../utils/colors';
+import {DEFAULT_PROFILE_PIC_PATH, SOURCE_MY_PROFILE} from '../../utils/constants';
+import dimen from '../../utils/dimen';
 import {fonts} from '../../utils/fonts';
+import {useUpdateClientGetstreamHook} from '../../utils/getstream/ClientGetStram';
 import {linkContextScreenParamBuilder} from '../../utils/navigation/paramBuilder';
 import {requestCameraPermission, requestExternalStoragePermission} from '../../utils/permission';
-import {setFeedByIndex} from '../../context/actions/feeds';
-import {setMyProfileAction} from '../../context/actions/setMyProfileAction';
-import {setMyProfileFeed} from '../../context/actions/myProfileFeed';
-import {useAfterInteractions} from '../../hooks/useAfterInteractions';
-import {useUpdateClientGetstreamHook} from '../../utils/getstream/ClientGetStram';
-import {withInteractionsManaged} from '../../components/WithInteractionManaged';
-import ShadowFloatingButtons from '../../components/Button/ShadowFloatingButtons';
-import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
+import ShareUtils from '../../utils/share';
 import StorageUtils from '../../utils/storage';
+import RenderItem from '../FeedScreen/RenderList';
+import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
+import AnonymousTab from './elements/AnonymousTab';
+import BioAndDMSetting from './elements/BioAndDMSetting';
+import BottomSheetBio from './elements/BottomSheetBio';
+import BottomSheetImage from './elements/BottomSheetImage';
+import BottomSheetRealname from './elements/BottomSheetRealname';
+import FollowInfoRow from './elements/FollowInfoRow';
+import LinkAndSocialMedia from './elements/LinkAndSocialMedia';
+import ProfileHeader from './elements/ProfileHeader';
+import ProfilePicture from './elements/ProfilePicture';
+import ProfileTiktokScroll from './elements/ProfileTiktokScroll';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order has been changed and some imports have been removed. Make sure these changes do not affect the functionality of the code.
```diff
- import {colors} from '../../utils/colors';
- import {deleteAnonymousPost, deletePost} from '../../service/post';
- import {downVote, upVote} from '../../service/vote';
- import {fonts} from '../../utils/fonts';
- import {linkContextScreenParamBuilder} from '../../utils/navigation/paramBuilder';
- import {requestCameraPermission, requestExternalStoragePermission} from '../../utils/permission';
- import {setFeedByIndex} from '../../context/actions/feeds';
- import {setMyProfileAction} from '../../context/actions/setMyProfileAction';
- import {setMyProfileFeed} from '../../context/actions/myProfileFeed';
- import {useAfterInteractions} from '../../hooks/useAfterInteractions';
- import {useUpdateClientGetstreamHook} from '../../utils/getstream/ClientGetStram';
- import {withInteractionsManaged} from '../../components/WithInteractionManaged';
- import ShadowFloatingButtons from '../../components/Button/ShadowFloatingButtons';
- import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
- import StorageUtils from '../../utils/storage';
+ import {downVote, upVote} from '../../service/vote';
+ import {colors} from '../../utils/colors';
+ import {DEFAULT_PROFILE_PIC_PATH, SOURCE_MY_PROFILE} from '../../utils/constants';
+ import dimen from '../../utils/dimen';
+ import {fonts} from '../../utils/fonts';
+ import {useUpdateClientGetstreamHook} from '../../utils/getstream/ClientGetStram';
+ import {linkContextScreenParamBuilder} from '../../utils/navigation/paramBuilder';
+ import {requestCameraPermission, requestExternalStoragePermission} from '../../utils/permission';
+ import ShareUtils from '../../utils/share';
+ import StorageUtils from '../../utils/storage';
+ import RenderItem from '../FeedScreen/RenderList';
+ import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
+ import AnonymousTab from './elements/AnonymousTab';
+ import BioAndDMSetting from './elements/BioAndDMSetting';
+ import BottomSheetBio from './elements/BottomSheetBio';
+ import BottomSheetImage from './elements/BottomSheetImage';
+ import BottomSheetRealname from './elements/BottomSheetRealname';
+ import FollowInfoRow from './elements/FollowInfoRow';
+ import LinkAndSocialMedia from './elements/LinkAndSocialMedia';
+ import ProfileHeader from './elements/ProfileHeader';
+ import ProfilePicture from './elements/ProfilePicture';
+ import ProfileTiktokScroll from './elements/ProfileTiktokScroll';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->